src/mbtiles.cpp

```diff
@@ -304,13 +257,17 @@ ChartMBTiles::ChartMBTiles()
       pfc->Read ( _T ( "DebugMBTiles" ),  &m_b_cdebug, 0 );
 #endif
       m_pDB = NULL;
-
 }
 
 ChartMBTiles::~ChartMBTiles()
 {
+    std::cout << "closing tileset " << m_FullPath << std::endl;
```

Comment:
`std::cout` or logging?

Reply:
#1232 offers a solution here in the form of C++ -style logging classes/destinations...

Reply:
Don't pay much attention to this, it is there just to see how badly this affects the memory management and will never make it into the final code.